charts/testkube-cloud-api/values.yaml

```diff
@@ -145,7 +145,7 @@ api:
       enabled: false
       config: {}
   # -- Configure which invitation mode to use (email|auto-accept): email uses SMTP protocol to send email invites and auto-accept immediately adds them
-  inviteMode: auto-accept
+  inviteMode: email
```

Comment:
You have to configure the SMTP server in this case, right?

Reply:
yes

Reply:
I'm not sure this is a good default, this comes from customer experience.